app.py

```diff
@@ -120,23 +104,27 @@ def __init__(self, scope: Construct, construct_id: str, **kwargs) -> None:
 db_secret_name = database.pgstac.secret.secret_name
 db_security_group = database.db_security_group
 
+base_api_url = f"https://{veda_app_settings.stage_name()}.{veda_app_settings.veda_custom_host}".strip(
```

Comment:
Doesn't `VEDA_CUSTOM_HOST` already include the stage name?

Reply:
Ah it does!

Reply:
https://github.com/NASA-IMPACT/veda-backend/pull/422/commits/f0e088488e59f94207e427822f127965a0a5db11